### ISSUE TITLE
Training using PyTorch training loop

### ISSUE THREAD

**Issue by BardiaKh** [havakv/pycox#117]
Thank you for developing such a useful package and also maintaining it over time. I was wondering if there is a way to use the loss functions as vanilla loss functions in a pytorch training loop? I'm asking this because I have a previously implemented pipeline for classification and wanted to see if I could integrate your packages into it. I mean bypassing wrapping the `nn.Module` with one of the pycox methods.

**Comment by havakv** [comment]
Hi, and thank you for the kind words. So the branch [refactor_out_torchtuples](https://github.com/havakv/pycox/tree/refactor_out_torchtuples) addresses this. Hopefully, we should be able to merge this into the master branch at some point. This is also addressed in #25 and #60
